### ISSUE TITLE
Documentation Related Question

### ISSUE THREAD

**Issue by merianos** [vexelon-dot-net/e-additives.server#18]
Hello @petarov.

Start by saying a big thank you for this API. It seems to be very promising and useful tool.

Know my question.

I just read the documentation, and I found this response:

```
{
    "id": "1",
    "code": "100",
    "last_update": "2013-09-08T20:16:36+0300",
    "name": "Curcuma (turmeric)",
    "status": "Safe",
    "veg": null,
    "function": "food coloring (yellow-orange)",
    "foods": "Fats and oils, curry, processed cheese and fish fingers.",
    "notice": "Safe in recommended doses. High doses should not be taken by people with gallstones, obstructive jaundice, acute bilious colic or toxic liver disorders.",
    "info": "Orange-yellow colouring extracted from the rhizomes of the turmeric plant.",
    "url": "http://hostname/additives/100"
}
```

Of course, most of the fields are self-explained based on the name you have assigned to them, but there are two fields with no meaningful names, and those two fields are the `status` and the `veg`.

I guess the `veg` field` it is used as a flag for vegetarians safe foods, but I am not really sure if that's the purpose of that field.

In addition, I am not sure if the `status` is always right, or, if it has a different meaning than what I understand. The reason I ask that is that when I access your website and see the additive [`E 120`](http://e-additives.vexelon.net/index.html#!/additives/120), you have set the status to empty string, while the warning says that is better to now consumed. In other cases, such as the [`E 120`](http://e-additives.vexelon.net/index.html#!/additives/121) you make it clear that this additive is not safe at all.

So what is the purpose of the attribute `veg` and `status`?

Thank you in advance! :smiley:

**Comment by petarov** [comment]
Hi!

Thanks! Those are indeed good questions. This is the best I can recall at the moment:

* `status` - An all purpose text field about whether the particular additive is forbidden or not. It might for example, be forbidden just in France or in the US, but allowed everywhere else. 
* `veg` - This was originally meant as indicator for vegetarian safety, however, I think there is not enough data in the database to rely on the field, hence why I never put it to real use.  

To summarize the `status` and `notice` fields are somewhat complementary. An empty `status` should mean the additive isn't forbidden anywhere, but `notice` may indeed show warnings about it. As it turns out, even if an additive is found (by some studies) as dangerous, it is does not necessarily get forbidden. 

In any case, be advised that this data is more than decade old now. Things might have changed. 😉

**Comment by merianos** [comment]
Wow !! :) That was really too detailed answer !! :) xaxaxa

Thank you so much for your feedback and keep in mind your [live version](http://e-additives.vexelon.net/index.html#!/home) isn't safe.

I mean your data can be accessed without authentication :cry: 

I gave it a try and I found relatively easy your authentication information and I got some access to your API.

Take care of it, if still interested in this project.

Kind regards
Merianos Nikos
